### PR TITLE
[#6486] budgeting-list/tiles: adding reference number to tile

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

**Comment by khamui** [comment]
just saw gh actions failing, need to fix tests

File: meinberlin/apps/budgeting/serializers.py [hunk 1]
```diff
@@ -122,3 +123,6 @@ def get_vote_allowed(self, proposal):
             return has_voting_permission and is_three_phase_budgeting
 
         return False
+
+    def reference_number(self):
+        return '{:d}-{:05d}'.format(self.created.year, self.pk)
```
Comment (fuzzylogic2000) [inline meinberlin/apps/budgeting/serializers.py:128]: Cool! But now that we annotate it in the api (@Rineee s PR), I think you can just use that. 
Hmm, but maybe this is more robust as we don't need to add fallback when it's not there like for the comment_count and so on.

Comment (khamui) [reply meinberlin/apps/budgeting/serializers.py:128]: i will leave it for now, that might be not correct for future non-zero leading numbers probably?
Just fixed the tests for now

Comment (goapunk) [reply meinberlin/apps/budgeting/serializers.py:128]: I'm generally fine with both, just one thing to keep in mind: having two different sources for the reference number might be confusing? so using the annotation might avoid some confusion later...and it means just changing the function to
```
def reference_number(self):                                                 
          if hasattr(proposal, 'ref_number':                                      
              return proposal.ref_number                                          
          else:                                                                   
              return ''
```

**Review by goapunk — APPROVED** [review]
LGTM, just a comment regarding the annotation

File: meinberlin/apps/budgeting/serializers.py (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/budgeting/serializers.py:128]: yes, I was with "not that important now", but @goapunk is right, one change now, but will be less confusion later and we will notice when it's not annotated. :)

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**